src/celestia/celestiacore.cpp

```diff
@@ -90,6 +90,47 @@ static const double OneFtInKm = 0.0003048;
 static const double OneLbInKg = 0.45359237;
 static const double OneLbPerFt3InKgPerM3 = OneLbInKg / pow(OneFtInKm * 1000.0, 3);
 
+namespace
+{
+float KelvinToCelsius(float kelvin)
+{
+    return kelvin - 273.15f;
+}
+
+float KelvinToFahrenheit(float kelvin)
+{
+    return kelvin * 1.8f - 459.67f;
+}
+
+FormattedNumber SigDigitNum(double v, int digits)
+{
+    return FormattedNumber(v, digits,
+                           FormattedNumber::GroupThousands |
+                           FormattedNumber::SignificantDigits);
+}
+
+string KelvinToStr(float value, int digits, CelestiaCore::TemperatureScale temperatureScale)
+{
+    const char* unitTemplate = "";
+
+    switch (temperatureScale)
+    {
+        case CelestiaCore::Celsius:
+            value = KelvinToCelsius(value);
+            unitTemplate = _("{} °C");
```

Comment:
do these strings need localization?

Reply:
> do these strings need localization?

Usually they don't need it. Those units are internationally recognized anyway.

Reply:
removed localization